test/e2e/create_test.go

```diff
@@ -293,15 +301,6 @@ func (tc *testContext) waitForWindowsNodes(nodeCount int32, expectError, checkVe
 	// not take the number of nodes into account. If we are testing node creation without applying Windows label, we
 	// should throw error within 5 mins.
 	err = wait.Poll(nodeRetryInterval, time.Duration(math.Max(float64(nodeCount), 1))*creationTime, func() (done bool, err error) {
-		// TODO: vSphere CSRs are not being approved due to a hostname mismatch. This is an issue only because the
-		//       ConfigMap controller does not have the ability to approve CSRs yet. This needs to be removed when that
-		//       functionality is added in https://issues.redhat.com/browse/WINC-581
-		if tc.CloudProvider.GetType() == config.VSpherePlatformType {
-			if err := tc.approveAllCSRs(); err != nil {
-				log.Printf("error approving CSRs: %s", err)
-				return false, nil
-			}
-		}
```

Comment:
Removing these changes in a separate commit would make this easier to read, will give the tests a better look after that

Reply:
Why? This commit is introducing changes that requires a change in the e2e tests. Question to ask is, can I revert this individual commit?